packages/web-components/fast-element/src/observation/observable.ts

```diff
@@ -1,6 +1,7 @@
 import { DOM } from "../dom";
 import { Notifier, PropertyChangeNotifier, SubscriberSet, Subscriber } from "./notifier";
 
+const volatileRegex = /(\?|\&\&|if)/;
```

Comment:
should this be testing for `x.foo || x.bar` instead of `x.foo && x.bar`?

Reply:
Hmm. Probably need to test both actually. Because if the first term of an `&&` is false, it won't access the second but if the first term of an `||` is true, it also wouldn't access the second. I'll fix that.

Reply:
ahh yea you're right - `&&` won't always access both.

Reply:
oh hmm - you probably also want to catch switch cases huh?

Reply:
Fixed and added tests for `||` and `switch/case`.